boards/adafruit-clue/doc.txt

```diff
@@ -0,0 +1,43 @@
+/**
+@defgroup    boards_adafruit-clue Adafruit Clue
+@ingroup     boards
+@brief       Support for the Adafruit Clue
+
+### General information
+
+The [Adafruit Clue](https://www.adafruit.com/clue) board
+is an opensource, micro development kit using the nRF52840 SoC.
+This board provides 802.15.4 and BLE connectivity.
+
+<img src="https://raw.githubusercontent.com/adafruit/Adafruit-CLUE-PCB/master/assets/4500.jpg"
+     alt="Adafruit Clue" style="height:800px;"/>
+
+### Schematics
+
+The board detailed description and schematic is available
+[here](https://github.com/adafruit/Adafruit-CLUE-PCB).
+
+### Flash the board
+
+The board is flashed using the `nrfutil` Python package.
```

Comment:
```suggestion
The board is flashed using the `nrfutil` Python package:
    $ pip install nrfutil --user
```

Reply:
Addressed, note that the doc was wrong, the package to install is adafruit-nrfutil.